app/views/impact/PageImpact/index.vue

```diff
@@ -179,7 +179,13 @@
       .col-lg-12
         a.btn.btn-primary.btn-lg(href="/teachers/quote") {{ $t('general.contact_us') }}
 
-  ozaria-stats
+  .row.row-dark#ozaria-efficacy
+    .row.width-container
+      .col-md-12
+        h2.text-gold.text-center {{ $t('impact.ozaria_impact_title') }}
+        h3.text-h3.text-center.text-white {{ $t('impact.ozaria_impact_blurb') }}
```

Comment:
Let's shrink the width on this `h3` so that it's not quite as wide (easier reading) and to the point where it just so happens to have `"Learn more about Ozaria's curriculum efficacy here."`(which will be a hyperlink) start on the third line when viewed at normal desktop size.